Makefile.toml

```diff
@@ -1039,6 +1045,151 @@ ln -sfn "${PUBLISH_REPO_OUTPUT_DIR##*/}" "${PUBLISH_REPO_OUTPUT_DIR%/*}/latest"
 '''
 ]
 
+# This is a bash setup right now just to enable my testing
+# We should move this into the pubsys repo logic now
+[tasks.import-images]
+dependencies = ["publish-setup", "publish-tools", "tuftool"]
+script = [
+'''
+set -e
+
+export PATH="${BUILDSYS_TOOLS_DIR}/bin:${PATH}"
+
+IMAGE="bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${BUILDSYS_VERSION_BUILD}"
+OUTDIR="${BUILDSYS_OUTPUT_DIR}/${BUILDSYS_VERSION_BUILD}"
+
+bootlz4="${IMAGE}-boot.ext4.lz4"
+imglz4="${IMAGE}.img.lz4"
+rootlz4="${IMAGE}-root.ext4.lz4"
+hashlz4="${IMAGE}-root.verity.lz4"
+
+# TODO - do we care about these? At the moment I don't but probably later I do - we could get the manifest.json and fetch them there
+# kmodtarxz="${IMAGE}-kmod-kit.tar.xz" #  Versioned with v
+# migrationstar="${IMAGE}-migrations.tar" # not in repo
+# datalz4="${IMAGE}-data.img.lz4" # This doesn't seem to exist in the repo
+
+# Here is the list of files actually built, from there, its up to me to figure out what ones need to be solved for *Version without v
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${VERSION}-${HASH}-boot.ext4.lz4
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${VERSION}-${HASH}-data.img.lz4
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${VERSION}-${HASH}.img.lz4
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${VERSION}-kmod-kit.tar.xz
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-v${VERSION}-${HASH}-migrations.tar
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${VERSION}-${HASH}-root.ext4.lz4
+#bottlerocket-${BUILDSYS_VARIANT}-${BUILDSYS_ARCH}-${VERSION}-${HASH}-root.verity.lz4
+
+if [ -s "${bootlz4}" ] || [ -s "${imglz4}" ] || [ -s "${rootlz4}" ] || [ -s "${hashlz4}" ]; then
+   echo "Image files already exist for the current version/commit - ${IMAGE} - please run 'cargo make clean' to remove them" >&2
```

Comment:
Maybe we should have a clean-images task so we don't blow out all the docker and rpm stuff.

Reply:
Yeah, that seems reasonable. I was focusing on CI/CD flows but it stands to reason one might have other builds in their directory that they don't want cleaned.